packages/zosfiles/src/methods/create/Create.ts

```diff
@@ -134,7 +134,10 @@ export class Create {
         }
     }
 
-    public static async dataSetLike(session: AbstractSession, dataSetName: string, likeDataSetName: string): Promise<IZosFilesResponse> {
+    public static async dataSetLike(session: AbstractSession,
+                                    dataSetName: string,
+                                    likeDataSetName: string,
+                                    options?: Partial<ICreateDataSetOptions>): Promise<IZosFilesResponse> {
         // Required
         ImperativeExpect.toNotBeNullOrUndefined(dataSetName, ZosFilesMessages.missingDatasetName.message);
         ImperativeExpect.toNotBeNullOrUndefined(likeDataSetName, ZosFilesMessages.missingDatasetLikeName.message);
```

Comment:
Please validate the options before calling the ZosmfRestClient. Example of doing this in `Create.dataSet`:
https://github.com/zowe/zowe-cli/blob/6b993c90093e84ab3af7a0e281fda7d8b86fdff7/packages/zosfiles/src/methods/create/Create.ts#L123-L125